docs/configuration/indexes.md

```diff
@@ -211,6 +211,25 @@ credentials cannot be found. If the discovered credentials are not valid (i.e.,
 HTTP 401 or 403), then uv will treat packages as unavailable and query the next configured index as
 described in the [index strategy](#searching-across-multiple-indexes) section.
 
+### Ignoring error codes when searching across indexes
+
+When using the `first-index` strategy, uv will stop searching if it encounters a `401 Unauthorized`
+or `403 Forbidden` response status code. The one exception is that uv will ignore 403s when
+searching the `pytorch` index (since this index returns a 403 when a package is not present).
+
+Users can configure which error codes are ignored for an index, using the `ignored-error-codes`
```

Comment:
We avoid referring to "Users" like this in the documentation. Instead, I'd say "You can configure which error codes..." or, more typically, "To ignore additional error codes for an index, use the `...` setting."